client/dist/style.css

```diff
@@ -1,7 +1,162 @@
-@import "reset.css";
-@import "fonts.css";
+/* @import "reset.css"; */
+@font-face {
+  font-family: "Noto Regular";
+  src: url("./res/NotoSans-Regular.ttf");
+}
+@font-face {
+  font-family: "Noto Bold";
+  src: url("./res/NotoSans-Bold.ttf");
+}
+@font-face {
+  font-family: "Noto Italic";
+  src: url("./res/NotoSans-Italic.ttf");
+}
+@font-face {
+  font-family: "Noto Bold Italic";
+  src: url("./res/NotoSans-BoldItalic.ttf");
+}
+html {
+  font-size: 16px;
+}
+
+button {
```

Comment:
This will depend on how we end up combining all the components, but it may be best to avoid styling based on element name alone (i.e. classes and/or id's for each specific button/set of buttons may be necessary for each component)

Reply:
100% correct, my bad